src/command_line/indexing/create_index.rs

```diff
@@ -0,0 +1,53 @@
+pub use super::prelude::*;
+use crate::command_line::indexing::split_path;
+use crate::indexing::index_manager::IndexManager;
+use std::ops::Deref;
+
+pub const NAME: &str = "create-index";
+
+pub fn command() -> Command {
+    Command::new(NAME)
+        .arg(
+            Arg::new("index_path")
```

Comment:
I think dashes are good here. But again, please confirm the other bits of the codebase use dashes 😆